build/cloudbuild.yaml

```diff
@@ -25,12 +25,12 @@ steps:
   args: [fetch, --depth=100]
 - name: "ubuntu"
   args: ["mkdir", "-p", "/workspace/_output/linux_amd64"]
-- name: "gcr.io/kubebuilder/thirdparty-linux:1.10"
+- name: "gcr.io/kubebuilder/thirdparty-linux:1.10.1"
```

Comment:
I am assuming we already have thirdparty-linux:1.10.1 already in gcr ? otherwise all build triggers will fail.
If not, then one way to address it would be to have thirdparty tools docker-file and cloudtools.yml file committed in a separate check-in and have thirdparty image 1.10.1 ready and then check in the rest.

Reply:
We already have this.  I pushed it :)